### PR TITLE
Add yAxisBaseline to the stacked-area chart

### PR DESCRIPTION
**Pull request by ajdani**
## Description
Adds the functionality to set the yAxisBaseline to a custom-value for the stacked-area chart. This baseline is the y-value where the stacked area is starting from.

Setting `.yAxisBaseline(-20)` sets the baseline to **-20** and changes the _minY_ of the scale as well to -20.

## How Has This Been Tested?
- Test that the maxY is set to baseline if the baseline is bigger then the max of the data
- Test that the minY is set to baseline if the baseline is smaller then the min of the data
- Test the getter / setter of the yAxisBaseline

tbh there is no test which ensures that the drawing is correct, e.g. the baseline changes. I did not have an idea how to tackle this. Maybe you can provide some ideas please?

## Screenshots (if appropriate):
image

## Types of changes
- [ ] Refactor (changes the way we code something without changing its functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] Latest master code has been merged into this branch
- [x] No commented out code (if required, place // TODO above with explanation)
- [ ] No linting issues
- [x] Build is successful
- [x] Code follows the [API Guidelines](http://eventbrite.github.io/britecharts/topics-index.html#toc5__anchor)
- [ ] Updated the documentation
- [x] Added tests to cover changes
- [x] All new and existing tests passed

### CODE REVIEW

File: src/charts/stacked-area.js [hunk 1]
```diff
@@ -1561,6 +1584,22 @@ define(function(require){
             return this;
         };
 
+        /**
+         * Gets or Sets the yAxisBaseline
```
Comment (DalerAsrorov) [inline src/charts/stacked-area.js:1588]: This is the part that goes into API docs. Could you add a more detailed description of what it does here? In addition, providing an example here would be super helpful as well.

Comment (DalerAsrorov) [reply src/charts/stacked-area.js:1588]: You can document an example of code like this:
https://github.com/eventbrite/britecharts/blob/master/src/charts/stacked-area.js#L1519

Comment (ajdani) [reply src/charts/stacked-area.js:1588]: Thanks for the hint how to document examples. Will add it.

**Comment by ajdani** [comment]
Thanks for the review @DalerAsrorov !

Added some documentation and an example.

**Pull request merged**